package.json

```diff
@@ -40,8 +40,7 @@
     "react-keydown": "^1.9.7",
     "sequelize": "^4.38.0",
     "sqlite3": "^4.0.1",
-    "style-loader": "^0.21.0",
-    "winston": "^3.0.0"
+    "style-loader": "^0.21.0"
```

Comment:
the `ˆ` can install a major version with break changes which can impact the code.